lib/web_of_science/map_names.rb

```diff
@@ -88,18 +88,28 @@ def medline_name(name)
         name[:given_name] = given
         name[:first_name] = first if first.present?
         name[:middle_name] = middle if middle.present?
+        name[:name] = "#{name[:last_name]},#{name[:first_name]},#{name[:middle_name]}" # full name in the older style format used by Profiles/CAP
```

Comment:
Are you OK w/ the possibility that this becomes `'Anonymous,,'`?  That seems inelegant, at best.

Reply:
Yeah, its not great, but this is the format they are parsing on their end for display.  Since first and middle name are blank in this case, I'd rather leave it like that and let their parsing decide what to do (they must already handle cases of blank middle names anyway).

Reply:
OK, nevermind, I see you tested exactly that below.